src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto

```diff
@@ -763,8 +771,8 @@ message FindRelatedBooksResponse {
 
 // Metadata for a longrunning operation.
 message GetBigBookMetadata {
-    // Approximate percentage of the book processed thus far.
-    int32 progress_percent = 1;
+  // Approximate percentage of the book processed thus far.
```

Comment:
The rest of the diff is just to remove the 4-space indentation that had been bugging me.